.changes/fix-nsis-webview2-installer-use-tempdir.md

```diff
@@ -0,0 +1,5 @@
+---
+'tauri-bundler': 'patch:bug'
+---
+
+On Windows, fix NSIS installer writing webview2 installer file to the well-known temp dir instead of the install dir, so we don't pollute the install dir.
```

Comment:
```suggestion
---
'tauri-bundler': 'patch:enhance'
---

On Windows, NSIS installer will write webview2 installer file to the well-known temp dir instead of the install dir, so we don't pollute the install dir.
```